js/private/test/node-patches/BUILD.bazel

```diff
@@ -17,9 +17,7 @@ TESTS = [
             "//js/private/node-patches/src:compile",
         ],
         entry_point = t,
-        env = {
-            "JS_BINARY__DISABLE_NODE_PATCHES": "1",
```

Comment:
breaking I guess, I think I used this somewhere and will find out :)

Reply:
it wasn't advertised anywhere and I didn't use it anywhere else so should be fine